tests/__init__.py

```diff
@@ -110,8 +113,15 @@ def test_func(self):
                         msg=f"The actual value for .{key}() did not match the expected value.",
                     )
 
-        # Assert that the ingredients returned by the ingredient_groups() function
-        # are the same as the ingredients return by the ingredients() function.
+        # Special handling for ratings_count
+        if "ratings_count" in expect.keys():
+            with self.subTest("ratings_count"):
+                ratings_count = actual.ratings_count()
+                self.assertTrue(
+                    expect["ratings_count"] >= ratings_count,
```

Comment:
Hm, in fact I think there's still a problem here - shouldn't this be:

```suggestion
                    ratings_count >= expect["ratings_count"],
```

?

In other words: ratings count scraped from the `.testhtml` (that may have been updated during maintenance/bugfixes) is greater-than-or-equal-to the `ratings_count` in the `.json` test data (that may have been partially-updated during hypothetical maintenance/bugfixes).

Reply:
You're completely right, I was testing the opposite way. I'll test this again later before updating